Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/FitGaussianPeaks.py

```diff
@@ -17,6 +17,8 @@ class FitGaussianPeaks(DataProcessorAlgorithm):
     _estimate_sigma = 3.0
     _min_sigma = 0.0
     _max_sigma = 30.0
+    _estimate_fit_window = True
+    _fit_window = 5
```

Comment:
I'm guessing this is the index of the fitting window, or similar?
Could we add a comment explaining what this is as it's not immediately obvious